content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -2,6 +2,10 @@
 title: Managing Compute Resources for Containers
 content_template: templates/concept
 weight: 20
+feature:
```

Comment:
What does the `description: >` attribute affect, and why is it implemented in this PR?

Please say more about what's happening here. The description text doesn't appear anywhere on the page, nor do similar additions in other files render visible results.

Reply:
![image](https://user-images.githubusercontent.com/394382/45286833-1ce4c580-b4e7-11e8-920d-bb35413381ef.png)

Reply:
See the `{{< blocks/kubernetes-features >}}` shortcode.

Reply:
Got it. 👍